run_tests.py

```diff
@@ -1,5 +1,6 @@
 
 import unittest
+import vcr
```

Comment:
Import VCR is used as far as I can tell..

Bill Cromie
t:917.215.3576

sent on the move

On Jan 23, 2015 7:28 PM, "Alex" <notifications@github.com> wrote:

> In run_tests.py
> <https://github.com/aberke/check-list/pull/6#discussion_r23490664>:
>
> > @@ -1,5 +1,6 @@
> >
> >  import unittest
> > +import vcr
>
> used?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aberke/check-list/pull/6/files#r23490664>.
>